java/src/jmri/jmrit/display/controlPanelEditor/shape/DrawFrame.java

```diff
@@ -146,9 +162,6 @@ private final JPanel makePanel() {
         _lineColorButon.setSelected(true);
         panel.add(p);
         _chooser = new JColorChooser(Color.LIGHT_GRAY);
-        _chooser.setColor(Color.green);
-        AbstractColorChooserPanel _chooserColorPanels[] = { new ButtonSwatchColorChooserPanel()};
```

Comment:
Please restore the ButtonSwatchColorChooserPanel.

See @rhwood's comments on #4342